charts/rasa-x/values.yaml

```
@@ -20,6 +20,12 @@ rasax:
   databaseName: "rasa"
   # disableTelemetry permanently disables telemetry
   disableTelemetry: false
+  # initialUser is the user which is created upon the initial start of Rasa X
+  initialUser:
+    # username specifies a name of this user
+    username: "admin"
```

how about having the default community user here?
```suggestion
    username: "me"
```

---

👍